app/Http/Controllers/HomeController.php

```diff
@@ -19,12 +19,35 @@
  */
 namespace App\Http\Controllers;
 
+use App\Models\BanchoStats;
+use App\Models\Count;
+use Carbon\Carbon;
+use Auth;
 use View;
 
 class HomeController extends Controller
 {
     protected $section = 'home';
 
+    public function getLanding()
+    {
+        if (Auth::check()) {
+            return $this->getNews();
+        }
+
+        $timeAgo = Carbon::now()->subHours(24);
+        $stats = BanchoStats::where('date', '>=', $timeAgo)
+            ->whereRaw('banchostats_id mod 10 = 0')
+            ->get();
+        $totalUsers = Count::totalUsers();
+        $currentOnline = (is_null($stats) ? 0 : $stats->last()->users_osu);
```

Comment:
`$stats` will never be null. It'll just be an empty array if there's no matching stats.